config/routes/api.rb

```diff
@@ -7,7 +7,7 @@
 
     get 'env_settings', to: 'env_settings#index'
 
-    patch 'locale', to: 'locale#update'
+    patch 'profile', to: 'profile#update'
```

Comment:
`resource :profile, only: [:update]` should also do the trick. A bit more consistent with the rest